realsense2_camera/launch/rs_launch.py

```diff
@@ -27,56 +27,57 @@
                            {'name': 'usb_port_id',                  'default': "''", 'description': 'choose device by usb port id'},
                            {'name': 'device_type',                  'default': "''", 'description': 'choose device by type'},
                            {'name': 'config_file',                  'default': "''", 'description': 'yaml config file'},
-                           {'name': 'unite_imu_method',             'default': "0", 'description': '[0-None, 1-copy, 2-linear_interpolation]'},
                            {'name': 'json_file_path',               'default': "''", 'description': 'allows advanced configuration'},
+                           {'name': 'initial_reset',                'default': 'false', 'description': "''"},
+                           {'name': 'rosbag_filename',              'default': "''", 'description': 'A realsense bagfile to run from as a device'},
                            {'name': 'log_level',                    'default': 'info', 'description': 'debug log level [DEBUG|INFO|WARN|ERROR|FATAL]'},
-                           {'name': 'output',                       'default': 'screen', 'description': 'pipe node output [screen|log]'},
```

Comment:
Deleted:
- output

Reply:
I think we can still use this parameter in the rs_launch.py, instead of putting "screen" hardcoded... you can read about this and update me..
I think using output:=log (if we change this in the rs_launch.py) can make save more logs in the logs directory..